src/vivarium_nih_us_cvd/data/loader.py

```diff
@@ -277,6 +288,19 @@ def get_entity(key: Union[str, EntityKey]):
         "healthcare_entity": healthcare_entities,
     }
     key = EntityKey(key)
+
+    if key.name == "high_fasting_plasma_glucose":
+        entity = type_map[key.type][key.name]
+        entity.distribution = "ensemble"
+        entity.tmred = Tmred(
+            distribution="uniform",
+            min=scalar(4.884880066),
```

Comment:
Should these constants go somewhere?

Reply:
I copied these directly from gbd_mapping and these are going to get replaced with new GBD data so I think it's OK for now.